stable/gangway/templates/configmap.yaml

```diff
@@ -10,5 +10,10 @@ metadata:
 data:
   gangway.yaml: |
     {{- .Values.gangway | toYaml | nindent 4 }}
+    {{- if .Values.trustedCAEnabled }}
+    trustedCAPath: /gangway/rootca.crt
+  rootca.crt: |
+    {{ .Files.Get "rootca.crt" | nindent 4}}
```

Comment:
I think, rather than forcing the user of the chart to make the ca in a file named `rootca.crt` in the current directory, it would be nicer to let them provide the path. Changing the value from a bool to a string. For example:

```
    {{ .Files.Get .Values.trustedCAEnabled | nindent 4}}
```
Works if you provide:

```
trustedCAEnabled: /tmp/some/path/ca.crt
```

And the if on L13 still works.

Reply:
No point in specifying full path because file must exist in chart directory.  Instead, I've changed to trustedCAFile: <filename>.  This still allows user to refer to a filename of their choice.

Reply:
Ahhh. I've never used `.Files.Get` I just assumed it read a file including path.

So I'm confused a now, if a user wants to install this chart and specify their own `trustedCAPath`, how does that work if you *have* to put the CA in the same directory as the chart itself?  The normal workflow for installing charts from this repo is `helm install stable/gangway`. But with this change they'd have to clone the repo and copy _their_ values and CA into this dir and install it with `helm install .` (or whatever). Is that right?

I'm not keen on breaking the ability to install from the stable repo just for a certificate. Some  other charts do this, (`prometheus-operator` and `postgres` specifically) but they do it for much larger files, like entire SQL dumps or large Grafana Dashboards.  It feels like for what is essentially a small chunk of text it would far nicer to the user to simply have them provide it in the input values. 

This wouldn't be the first chart to accept a cert via the `values.yaml`.

Reply:
Good point, I'm not sure if files in current dir where helm is invoked can be used by .Files.Get, but in any case, better to specify the cert in values.yaml and keep it a single source of truth for everything needed to deploy gangway.  Pls see latest commit.

Reply:
@rk295 any idea when you'll get the chance to review\merge this PR?

Reply:
I completely missed the notification that you had pushed more changes! Sorry. I'll look over lunch today.